repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl

```diff
@@ -25,7 +25,7 @@
                               This is part of
                               <#list article.apiservice as service>
                                 <@hst.link hippobean=service var="apiservice"/>
-                                <a temprop="url" href="${apiservice}" onClick="logGoogleAnalyticsEvent('document click','Event','${apiservice}');" onKeyUp="return vjsu.onKeyUp(event)" title="${service.title}"><span itemprop="name">${service.title}</span></a><#sep>,
+                                <a itemprop="url" href="${apiservice}" onClick="logGoogleAnalyticsEvent('document click','Event','${apiservice}');" onKeyUp="return vjsu.onKeyUp(event)" title="${service.title}"><span itemprop="name">${service.title}</span></a><#sep>,
```

Comment:
Related to API workstream, so could use "API" as the 2nd logGoogleAnalyticsEvent param?

Reply:
(y)

Reply:
solved by: https://github.com/NHS-digital-website/hippo/commit/4b1e9ee5a5497df9cca0f908a247b4bac8ad4d1a